src/main/java/net/dancier/domain/dance/School.java

```diff
@@ -0,0 +1,19 @@
+package net.dancier.domain.dance;
+
+import lombok.Builder;
+import lombok.Data;
+import net.dancier.domain.Recommendable;
+import net.dancier.domain.User;
+
+import java.net.URL;
+
+@Data
+@Builder
+public class School implements Recommendable {
+    private User user;
```

Comment:
Documentr the meaning of this user, please. What' the purpose of this field?

Reply:
I try to have my code selfdescribing ;-) (Not that I claim it is ;-) )
I renamed the field to owner, is this OK for you?

Reply:
Self describing code usually is an illusion ;-)

Reply:
By the way: owner would be great